docs/source/Rules/Rules.rst

```diff
@@ -326,15 +374,117 @@ It can also be used in rules. Every occurance of this text will then be replaced
 N.B. these references to task values only yield a value when the task is enabled and its value is valid.
 
 
+Event name (%eventname% or %eventpar%)
+--------------------------------------
+
+Added: 2022/04/17
+
+``%eventname%`` Substitutes the event name (everything before the '=' sign, 
+or the complete event if there is no '=' in the event)
+
+This can be useful for shortening the rules by matching events using a wildcard and then by using ``substring`` one may deduct the event origin.
+
+For example, trying to match events triggered by monitoring a number of pins on a GPIO extender.
+
+Typical events generated by the GPIO extenders look like this:
+
+.. code-block:: none
+  
+ EVENT: PCF#1=0
+ EVENT: PCF#2=0
+ ...
+
+Using ``%eventname%`` :
+
+.. code-block:: none
+
+ on PCF* do
+   logentry,"PCF pin: {substring:4:6:%eventname%} State: %eventvalue1%"
+ endon
+
+``%eventpar%`` is the part of ``%eventname%`` after the first ``#`` character.
+
+This allows to simplify the rules block described above:
+
+Using ``%eventpar%`` :
+
+.. code-block:: none
+
+ on PCF* do
+   logentry,"PCF pin: %eventpar% State: %eventvalue1%"
+ endon
+
 
 
 Event value (%eventvalue%)
 --------------------------
 
 Rules engine specific:
 
-%eventvalue% - substitutes the event value (everything that comes after
-the '=' sign, up to four values are possible).
+``%eventvalueN%`` - substitutes the N-th event value (everything that comes after
+the '=' sign).
+
+``%eventvalue%`` will be replaced by 
```

Comment:
This line seems not finished yet?

Reply:
I guess so ;)
Maybe someone just messaged me just when I was typing the documentation with some more good feedback on this issue ;)